vignettes/views.Rmd

```diff
@@ -20,7 +20,10 @@ A view is a view of all entities (File, Folder, Project, Table, Docker Repositor
 
 Preliminaries:
 ```{r collapse=TRUE}
+library(reticulate)
 library(synapser)
+syn <- reticulate::import("synapseclient")
+EntityViewType <- syn$EntityViewType
```

Comment:
```suggestion
library(synapser)
```

I don't think you need these - I was able to run through the vignette without it.

Reply:
So were we. The additional lines were added to satisfy the MacOS 12 runner where their absence led to build failures. Happy to discuss further.

Reply:
Interesting... Do you know what about the MacOS 12 runner cases these issues?  Can we add a comment about this?

Reply:
Interesting.. Can this be a comment?